plugins/redshift/dbt/include/redshift/macros/catalog.sql

```diff
@@ -147,7 +147,7 @@
         (sortkey_num > 0) as "stats:sortkey_num:include",
 
         'Approximate Size' as "stats:size:label",
-        size / 1000000.0 as "stats:size:value",
+        size * 1000000 as "stats:size:value",
```

Comment:
Should this really be `1048576` (1024*1024)?

Reply:
I goofed and accidentally removed my previous comment by cancelling a review...

That's a good question, and I had the exact same double take. The answer comes down to Megabytes vs. Mebibytes. [`SVV_TABLE_INFO` reports size in megabytes (MB)](https://docs.aws.amazon.com/redshift/latest/dg/r_SVV_TABLE_INFO.html). 1 Megabyte = 1,000,000 Bytes, whereas 1 Mebibyte (MiB) = 1,048,576 Bytes.